blocks/library/gallery/index.js

```diff
@@ -31,7 +31,7 @@ registerBlockType( 'core/gallery', {
 			type: 'array',
 			default: [],
 			source: 'query',
-			selector: 'div.wp-block-gallery figure.blocks-gallery-image img',
+			selector: 'ul.wp-block-gallery .blocks-gallery-item img',
```

Comment:
Is the classname change here intentional? `.blocks-gallery-item`? because this should reflect what's generated in the `save` function (not the `edit`)

Reply:
`.blocks-gallery-item` is intentional because it's now in `<li>` element, not in `<figure>` element. Therefore `.blocks-gallery-image` sounded weird for `<li>`.